recipes/pg8000/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pg8000" %}
+{% set version = "1.13.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eebcb4176a7e407987e525a07454882f611985e0becb2b73f76efb93bbdc0aab
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >3.5
```

The setup.py claims:
```suggestion
    - python >=3.5
```